src/constants/index.ts

```diff
@@ -2199,7 +2188,7 @@ export const IS_VIRTUAL_SWAP_ACTIVE = true
 export const IS_L2_SUPPORTED = true
 export const IS_SDL_LIVE = true
 export const IS_VESDL_LIVE = true
-export const IS_POOL_REGISTRY_MIGRATION_LIVE = false
+export const IS_POOL_REGISTRY_MIGRATION_LIVE = true
```

Comment:
not sure if this was intentional in this PR

Reply:
To remove `POOL_MAP` from BasicProvider, don't we need to get pools from registry contract?

Reply:
there should be no more usage off POOLS_MAP at this point.